docs/Node-for-Formula-Authors.md

```diff
@@ -72,6 +72,12 @@ This will install your Node module in npm's global module style with a custom pr
 bin.install_symlink Dir["#{libexec}/bin/*"]
 ```
 
+*Note:* If your module requires a (dev)dependencies to to present at npm prepublish time, you have to invoke `std_npm_install_args` with `prepublish_requires_deps` set to `true` like:
+
+```ruby
+system "npm", "install", *Language::Node.std_npm_install_args(libexec, prepublish_requires_deps = true)
```

Comment:
You can just do `std_npm_install_args(libexec, true)` here, since it's a default parameter and not a keyword (unless it gets changed above, in which case it will become `req_deps: true`).